### DISCUSSION TITLE
DeDRM and  acsm

### DISCUSSION THREAD

**Discussion by mario6097** [apprenticeharper/DeDRM_tools#1814] [General]
Hi,

~~does DeDRM help with something like URLLink.acsm?~~
Forget, I get the epub with Adobe Digital Edition.
Anyway, it does not seem I am able to deDRM my book, why?
thanks cheers
mario

**Comment by ElleKayEm** [comment]
Did you authorize Adobe Digital Editions with an Adobe ID?  Is it on the same machine and user as calibre with DeDRM?
